docs/learn/learn-comparisons.md

```diff
@@ -1,63 +1,189 @@
-therefore duplicate resources are required. Scalability on Polkadot is based on the security of the
-relay chain, and as the number of validators in the active set on Polkadot are increased, more
-parachains can be supported.
+In this section we explore the main differences in rollup technology between:
+
+- Zero-knowledge rollups,
+- Optimistic rollups, and
+- Polkadot rollups (i.e. parachains).
+
+Before diving into the differences between these rollup technologies, it is important to understand
+the difference between non-interactive and interactive methods.
+
+### Non-interactive vs. Interactive Rollup Methods
+
+In the context of rollups, **interactive** and **non-interactive** methods refer to how the validity
+of transactions or state changes is established between the rollup layer and the main blockchain
+(L1).
+
+A non-interactive method relies on cryptographic proofs to validate transactions or state changes
+without requiring dispute resolution or interactive challenges. A prominent example are
+zero-knowledge proofs such as a zk-SNARK or zk-STARK is generated off-chain, proving that the
+transactions in a batch satisfy the rules of the protocol. This proofs are submitted to the L1 chain
+and verified without needing further interaction.
+
+An interactive method involves a back-and-forth process between parties to validate transactions. A
+prominent example is fraud proofs, as used in optimistic rollups. If a dispute arises, a challenge
+mechanism is triggered, where a "prover" submits evidence (fraud proof) to demonstrate that a
+transaction is invalid. This can require multiple steps, with both the challenger and defender
+submitting data to the L1 chain.
+
+Here below, the key differences between interactive and non-interactive methods.
+
+| **Aspect**         | **Interactive**                                 | **Non-Interactive**                                                     |
+| ------------------ | ----------------------------------------------- | ----------------------------------------------------------------------- |
+| **Mechanism**      | Back-and-forth dispute resolution.              | One-time proof submission and verification.                             |
+| **Latency**        | Long finality due to challenge periods.         | Near-instant finality.                                                  |
+| **Security Basis** | Economic incentives and monitoring.             | Cryptographic guarantees.                                               |
+| **Complexity**     | Simpler to implement but requires monitoring.   | Computationally intensive and harder to build.                          |
+| **Use Cases**      | Broad compatibility with existing applications. | Best for high-security applications where instant finality is critical. |
+
+Both approaches serve different needs and trade-offs, depending on the balance between scalability,
+security, latency, and flexibility required.
```

Comment:
Might be nice to add a note about where ELVES falls here (Interactive, but assumes the "pessimistic" case)

Reply:
added, thank you!